experiment/dependencies/gomod_staleness.py

```diff
@@ -0,0 +1,151 @@
+ #!/usr/bin/env python3
```

Comment:
I think is complaining because you need to add boilerplate here 

> 1 files have incorrect boilerplate headers:

Reply:
fixed